### PR TITLE
Docker Desktop release notes 2.5.2

### PR DESCRIPTION
**Pull request by lorenrh**
Signed-off-by: Lorena Rangel <lorena.rangel@docker.com>

### Proposed changes

Docker Desktop release notes 2.5.2

### CODE REVIEW

File: docker-for-mac/edge-release-notes.md [hunk 1]
```diff
@@ -11,6 +11,23 @@ This page contains information about Docker Desktop Edge releases. Edge releases
...
+## Docker Desktop Community 2.5.2
+2020-11-26
+
+> [Download](https://desktop.docker.com/mac/edge/50165/Docker.dmg)
+
+### New
+
+- Use of three digit version number.
+- Starting with Docker Desktop 2.5.2, updates will be much smaller as they will be applied using delta patches.
+- Support for MacOS 11.0 (Big Sur) with Intel processors.
```
Comment (stephen-turner) [inline docker-for-mac/edge-release-notes.md:23]: This was already in 2.5.1.0

File: docker-for-mac/edge-release-notes.md [hunk 2]
```diff
@@ -11,6 +11,23 @@ This page contains information about Docker Desktop Edge releases. Edge releases
...
+
+### Bug fixes and minor changes
+
+- Re-enabled the experimental SOCKS proxy. Fixes [docker/for-mac#5048](https://github.com/docker/for-mac/issues/5048).
+- Fixed an unexpected EOF error when trying to start a non-existing container with `-v /var/run/docker.sock:`. See [docker/for-mac#5025](https://github.com/docker/for-mac/issues/5025).
+- Display an error message if the application needs write access on specific directories instead of crashing. See [docker/for-mac#5068](https://github.com/docker/for-mac/issues/5068)
```
Comment (stephen-turner) [inline docker-for-mac/edge-release-notes.md:29]: Change this sentence around: Display an error message instead of crashing if the ...

Comment (stephen-turner) [inline docker-for-mac/edge-release-notes.md:29]: Change this sentence around: Display an error message instead of crashing if the ...

**Review by stephen-turner — CHANGES_REQUESTED** [review]
Just a couple of small changes

File: docker-for-mac/edge-release-notes.md (hunk 1 above)
Comment (lorenrh) [reply docker-for-mac/edge-release-notes.md]: Removed!

File: docker-for-mac/edge-release-notes.md (hunk 2 above)
Comment (lorenrh) [reply docker-for-mac/edge-release-notes.md]: Done!

**Review by stephen-turner — APPROVED** [review]
Thanks, looks good now.

**Review by thaJeztah — APPROVED** [review]
LGTM, thanks!

**Pull request merged**